osgp/shared/shared/src/main/java/org/opensmartgridplatform/shared/security/RsaEncryptionService.java

```diff
@@ -103,7 +102,7 @@ public RsaEncryptionService() {
      * @param keyPair
      *            a key pair for algorithm {@value #ALGORITHM}.
      */
-    protected RsaEncryptionService(final KeyPair keyPair) {
+    protected RsaEncryptionService(final KeyPair keyPair) throws EncrypterException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'org.opensmartgridplatform.shared.exceptionhandling.EncrypterException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)